src/main/java/model/FruitTransaction.java

```diff
@@ -0,0 +1,53 @@
+package model;
+
+public class FruitTransaction {
+    private Operation operation;
+    private String fruit;
+    private int quantity;
+
+    public enum Operation {

```

Comment:
The Operation enum is well-defined. However, consider encapsulating the 'code' property more strictly by providing a getter method for it, rather than accessing it directly in the 'fromCode' method. This adheres to the encapsulation principle.